src/components/SelectedStudentsActionWrapper.js

```diff
@@ -1,8 +1,94 @@
 import React, {Component} from 'react';
+import Modal from 'react-modal';
 import { CSVLink } from 'react-csv';
+import { isEmpty } from 'simple-react-data-grid/src/utils/CommonUtils';
 
-class SelectedStudentsActionWrapper extends Component{
+const customPrintOptionStyles = {
+  overlay: {
+    zIndex: '999',
+    backgroundColor: 'rgba(21, 20, 20, 0.75)'
+  },
+  content: {
+    top: '50%',
+    position: 'absolute',
+    left: '50%',
+    right: 'auto',
+    bottom: 'auto',
+    border: '1px solid rgb(205, 68, 3)',
+    boxShadow: 'rgb(99, 99, 99) 0px 2.3px 3px 0px',
+    padding:'0px !important',
+    marginRight: '-50%',
+    width: '85%',
+    outline: 'none',
+    transform: 'translate(-50%, -50%)',
+    height: 'fit-content',
+  }
+};
 
+class SelectedStudentsActionWrapper extends Component{
+  constructor(props) {
+    super(props);
+    this.state = {
+      printOptionIsOpen:false,
+    }
+    this.openPrintOption = this.openPrintOption.bind(this);
+    this.closePrintOption = this.closePrintOption.bind(this);
+    this.renderStudentIdCards =this.renderStudentIdCards.bind(this);
+  };
+  openPrintOption() {
+    this.setState({printOptionIsOpen: true});
+  }
+  closePrintOption() {
+    this.setState({printOptionIsOpen: false});
+  }
+  renderStudentIdCards(student){
+   const studentsIdCards = student.map(object =>
+      <div className="student-Id-Cards">
```

Comment:
```suggestion
      <div className="student-id-cards">
```
Incorrect class name convention

Reply:
Done.